FinniversKit/Sources/DNA/Color/Color+SwiftUI.swift

```diff
@@ -63,14 +65,16 @@ extension Color {
 
     // MARK: - Others
 
-    public static var accentPrimaryBlue: Color   { Color(UIColor.accentPrimaryBlue)}
-    public static var accentSecondaryBlue: Color { Color(UIColor.accentSecondaryBlue) }
-    public static var accentPea: Color           { Color(UIColor.accentPea) }
-    public static var accentToothpaste: Color    { Color(UIColor.accentToothpaste) }
-    public static var tableViewSeparator: Color  { Color(UIColor.tableViewSeparator) }
-    public static var imageBorder: Color         { Color(UIColor.imageBorder) }
-    public static var decorationSubtle: Color    { Color(UIColor.decorationSubtle) }
-    public static var borderDefault: Color       { Color(UIColor.borderDefault) }
+    public static var accentPrimaryBlue: Color          { Color(UIColor.accentPrimaryBlue)}
+    public static var accentSecondaryBlue: Color        { Color(UIColor.accentSecondaryBlue) }
+    public static var accentPea: Color                  { Color(UIColor.accentPea) }
+    public static var accentToothpaste: Color           { Color(UIColor.accentToothpaste) }
+    public static var tableViewSeparator: Color         { Color(UIColor.tableViewSeparator) }
+    public static var imageBorder: Color                { Color(UIColor.imageBorder) }
+    public static var decorationSubtle: Color           { Color(UIColor.decorationSubtle) }
+    public static var borderDefault: Color              { Color(UIColor.borderDefault) }
+    public static var nmpBrandControlSelected: Color    { Color(UIColor.nmpBrandControlSelected) }
```

Comment:
Do we need the `nmpBrand` prefix?

Reply:
I don't think we do, I asked myself the same question when I saw it 🤔 Renaming it is out of scope for this PR and will make it a breaking change though. But perhaps I can add another public var without the prefix, so we can start using that one in upcoming PRs - instead of increasing usage of nmpBrand prefix (I will use controlSelected color in many more PRs to come).

Reply:
I meant renaming the UIColor, that one is already used other places in the app.

Reply:
We don't have a 1to1 mapping in Warp for these either, we are trying to map out all the usage of colors in FinniverseKit here https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=2070969738

And on this draft PR https://github.com/finn-no/FinniversKit/pull/1290/files#diff-659017700820fe4dcb3aaffab44230d2e872c22784cb38cd47264e6b89d94328R12 you can see what we wil probably use instead of these colors later on

Reply:
Okay, I see _all_ these colors will be deprecated once we introduce Warp. So to avoid doing any extra work that will be overwritten soon anyways, I'll just keep them as they are right now. Thanks for the input, @mohsen-schibsted 😄